alf/algorithms/actor_critic_algorithm.py

```diff
@@ -70,6 +74,10 @@ def __init__(self,
             debug_summaries (bool): True if debug summaries should be created.
             name (str): Name of this algorithm.
             """
+        actor_network = actor_network_ctor(
+            input_tensor_spec=observation_spec, action_spec=action_spec)
+        value_network = value_network_ctor(input_tensor_spec=observation_spec)
+
         super(ActorCriticAlgorithm, self).__init__(
             observation_spec=observation_spec,
```

Comment:
I feel that it's unnecessary to pass observation_spec to the base RLAlgorithm class.

Reply:
It doesn't hurt to leave it there. It was once useful.